readme/doc_resource.go

```diff
@@ -1180,18 +1181,18 @@ func (r *docResource) Schema(
 				Computed:    true,
 			},
 			"use_slug": schema.StringAttribute{
-				Description: "**Use with caution!** Create the doc resource by importing an existing doc by its slug. " +
+				MarkdownDescription: "**Use with caution!** Create the doc resource by importing an existing doc by its slug. " +
```

Comment:
fix rendering on TF docs. will eventually come back and clean this up on all the schemas.